dist/PublicLab.Editor.js

```diff
@@ -22132,35 +22132,36 @@ module.exports = PublicLab.Formatter = Class.extend({
 
     // functions that accept standard <data> and output form data for known services
     _formatter.schemas = {
- 
+
       "publiclab": function(data) {
- 
+
         var output = {};
- 
-        output.title              = data.title || null; 
-        output.body               = data.body  || null; 
- 
+
+        output.title              = data.title || null;
+        output.body               = data.body  || null;
+
         // we can remove this from server req, since we're authenticated
-        output.authenticity_token = data.token || null; 
- 
+        output.authenticity_token = data.token || null;
+        // Whether note will be draft or not
+        output.draft = data.draft || false;
```

Comment:
Awesome !!! how about introducing new lines aroud these 2 lines as it's looking bit of messy. Thanks @poppashingles .

Reply:
New lines added, thanks!